### ISSUE TITLE
benchalerts: post messages to GitHub PRs

### ISSUE THREAD

**Issue by austin3dickey** [conbench/conbench#769]
Adding a GitHub Check with a markdown summary to a GitHub commit is nice, but it doesn't notify anyone (especially on default-branch commits), so it's not very actionable. Users of `benchalerts` should be able to opt in to the App posting a message on the relevant PR that links to the new Check.

**Comment by jonkeane** [comment]
This might be scope creep (but also might be "just" adding a custom text field at the beginning too). One use case for something like this is before we start failing builds based on benchmark regression — we could comment on them for a little while tagging benchmark maintainers specifically to get a feel for what kinds of alert rates we would be getting.

So basically, have it so that one could configure the message to be something like "heads up @jonkeane and @austin3dickey this benchmark build would have alerted..." depending on some condition (e.g. it _would_ have been marked ❌  )

**Comment by austin3dickey** [comment]
Yes! I was thinking the same thing. I think #756 is similar to your request. 

In general, we'd like users to be able to configure different alerting logic based on a small set of common conditionals (e.g. `if any(regressions(z=5)) then comment()`; `if any(errors) then [send_check(fail), comment()]`). That would enable your request and #756 too.

I'll make a new issue. I think completing #770 is essential to enable this.
